src/jwks.cc

```diff
@@ -73,6 +76,18 @@ class EvpPkeyGetter : public WithStatus {
     return createEvpPkeyFromRsa(rsa.get());
   }
 
+  bssl::UniquePtr<EVP_PKEY> createEvpPkeyFromPkcs8(
+      const std::string& pkey_pem) {
+    bssl::UniquePtr<BIO> buf(BIO_new_mem_buf(pkey_pem.data(), pkey_pem.size()));
+    bssl::UniquePtr<EVP_PKEY> key(
+        PEM_read_bio_PUBKEY(buf.get(), nullptr, nullptr, nullptr));
+    if (key == nullptr) {
+      updateStatus(Status::JwksPemParseError);
```

Comment:
prefer to define a new status specific for PKCS

Reply:
Done